libs/cardano-ledger-core/testlib/Test/Cardano/Ledger/Core/Binary/CDDL.hs

```diff
@@ -94,15 +165,23 @@ pool_keyhash = "pool_keyhash" =:= hash28
 vrf_keyhash :: Rule
 vrf_keyhash = "vrf_keyhash" =:= hash32
 
---------------------------------------------------------------------------------
--- Crypto
---------------------------------------------------------------------------------
+mkHashSized :: Word64 -> Rule
+mkHashSized size = "hash" <> T.pack (show size) =:= VBytes `sized` size
 
 hash28 :: Rule
-hash28 = "hash28" =:= VBytes `sized` (28 :: Word64)
+hash28 = mkHashSized 28
 
 hash32 :: Rule
-hash32 = "hash32" =:= VBytes `sized` (32 :: Word64)
+hash32 = mkHashSized 32
+
+hash64 :: Rule
+hash64 = mkHashSized 64
+
+bytes80 :: Rule
+bytes80 = "bytes80" =:= VBytes `sized` (80 :: Word64)
+
+hash448 :: Rule
+hash448 = mkHashSized 448
```

Comment:
I don't think we have an hashes of this size :open_mouth: I think you mistaken it with KES signature, which has this size.
```suggestion
```